src/apps/subjects/tests/tests.py

```diff
@@ -197,6 +197,36 @@ async def applet_one_lucy_reviewer_with_subject(
     return applet_one
 
 
+@pytest.fixture
+async def applet_three_lucy_owner(session: AsyncSession, applet_three: AppletFull, lucy) -> AppletFull:
```

Comment:
looks redunddant as applet created by the same user. So owner and respondent roles should exist